depends/class-2.6.0-a_max.patch

```diff
@@ -43,10 +43,22 @@ diff -uNr class_public-2.6.0.old/source/background.c class_public-2.6.0/source/b
                                 pba->bt_size,
                                 pba->index_bi_a,
 -                               pba->a_today,
-+                               pba->a_max,
++                               pba->a_today, // get quantites at z=0
                                 &last_index,
                                 pvecback_integration,
                                 pba->bi_size,
+@@ -1607,8 +1607,9 @@
+              pba->error_message);
+
+   /* substitute last line with quantities today */
++  /* NOTE: comment out b/c last index refers to a_max, not a_today
+   for (i=0; i<pba->bi_size; i++)
+-    pData[(pba->bt_size-1)*pba->bi_size+i]=pvecback_integration[i];
```

Comment:
The last index on the interplation table must be a_max if you replace it with a_today all hell is wrong.. unless you force a_today == a_max, but that's not allowing going to the future -- in class convention a/a_today is what we usually call a.

Reply:
`all hell is wrong` == the variable of the table is no longer monotonic a decreases from a_max to a_today near the last item.